azerothcore/azerothcore-wotlk
# fix(DB/gameobject): Core Fragment should not despawn

## Changes Proposed:
-  Remove respawn time of object "Core Fragment" target of quest "Attunement to the Core"

## Issues Addressed:
- The issue is that currently the respawn timer is 10 minutes after it has been looted. This is a quest that can be done as groups, so the respawn time should be instant.
- Source: [Wowhead](https://classic.wowhead.com/quest=7487/attunement-to-the-core#comments)
- Source: [Youtube](https://www.youtube.com/watch?v=AzBAzjkFrE4#t=415)



## Tests Performed:
- Applied SQL, the despawning stops and it can be looted continuously as I believe it should.


## How to Test the Changes:


- First get the quest: .quest add 7487
- Get to BRD: .tele brd
- Get to The Molten Bridge - what I did: ".go c id 9017", jump down and hug the left wall, run ahead and you will see the bridge.
- Loot the "Core Fragment". Wait for it to respawn... It takes 10 minutes without this fix.
- Apply the SQL and restart the game server  - and it never despawns and can be looted continuously if you discard of the quest item you should have just picked up

## Known Issues and TODO List:



## Target Branch(es):








 
## How to Test AzerothCore PRs
 
When a PR is ready to be tested, it will be marked as **[WAITING TO BE TESTED]**.

You can help by testing PRs and writing your feedback here in the PR's page on GitHub. Follow the instructions here:

http://www.azerothcore.org/wiki/How-to-test-a-PR
